build-configs/integreat/mainImprint.ts

```diff
@@ -59,7 +57,7 @@ const mainImprint = `
 </p>
 <p><strong>Urheber- und Nutzungsrecht</strong></p>
 <p>Die durch die Seitenbetreiber erstellten Inhalte und Werke auf diesen Seiten unterliegen dem deutschen Urheberrecht. 
-Alle auf www.integreat.app befindlichen Inhalte sind unter der Creative Commons-Lizenz <a href="https://creativecommons.org/licenses/by/4.0/">CC BY-NC 4.0</a> lizenziert.
+Alle auf www.integreat.app befindlichen Inhalte sind unter der Creative Commons-Lizenz <a href="https://creativecommons.org/licenses/by/4.0/">CC BY 4.0</a> lizenziert.
```

Comment:
@dkehne should we change this to
```suggestion
Alle auf integreat.app befindlichen Inhalte sind unter der Creative Commons-Lizenz <a href="https://creativecommons.org/licenses/by/4.0/">CC BY 4.0</a> lizenziert.
```
We don't really use the `www.` subdomain anymore.

Reply:
yeah please change and maybe add a <a href> to https://integreat.app behind.